### ISSUE TITLE
The alias "org.eolang.jeo.opcode" is not used

### ISSUE THREAD

**Issue by yegor256** [objectionary/opeo-maven-plugin#410]
I'm getting this, when trying to process the XMIR generated by opeo 0.3.3:

```
[INFO] --- eo:0.39.0:xmir-to-phi (default-cli) @ hone ---
[INFO] Processing XMIR: org/eolang/hone/App.xmir (8Kb)
[ERROR] net.sf.saxon.style.XSLMessage@3cb76e2: Errors identified:
  , : The alias "org.eolang.jeo.opcode" is not used;
[ERROR] #fatalError(): Processing terminated by xsl:message at line 31 in fail-on-errors.xsl; SystemID: file:///org/eolang/parser/fail-on-errors.xsl; Line#: 31; Column#: 34
[ERROR] Processing terminated by xsl:message at line 31 in fail-on-errors.xsl
```

As I see in the XMIR, you don't use any `opcode` objects, while the `+alias org.eolang.jeo.opcode` is  still there. It seems, you should remove the alias if no `opcode` are in the code.

This is where it fails for me: https://github.com/objectionary/hone-maven-plugin/actions/runs/10501119853/job/29090616907

**Comment by yegor256** [comment]
@volodya-lombrozo please, help with this

**Comment by volodya-lombrozo** [comment]
@yegor256 Done. This should be fixed in the `0.3.5` version.

**Comment by yegor256** [comment]
@volodya-lombrozo thanks, fixed!
